scripts/debug_tools/prepare_all_cmd.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import subprocess
+
+
+def execute(cmd):
```

Comment:
Don't we already have a similar utility somewhere?

Reply:
Yes we have, but that is too specific. The only generic enough utility is `Popen` itself.